processors/src/SvtRawDataAnaProcessor.cxx

```diff
@@ -251,46 +261,156 @@ void SvtRawDataAnaProcessor::initialize(TTree* tree) {
 
 
 bool SvtRawDataAnaProcessor::process(IEvent* ievent) {
+    //std::cout<<"hello5"<<std::endl;
     Float_t TimeRef=-0.0;
     Float_t AmpRef=1000.0;
     double weight = 1.;int count1=0;int count2=0;
     long eventTime = evH_->getEventTime();
     
     //I AM DOING CLUSTER MATCHING HERE :)
-    //std::cout<<"Here is the eventTime"<<eventTime<<std::endl;
+    std::cout<<"Here is the eventTime"<<eventTime<<std::endl;
     //std::cout<<"Here is the TSBank Trigger Time"<<tsBank_->T<<std::endl;
     bool doClMatch = true;
     
-    
+    int STR = -10000;
```

Comment:
usually, variables are lower case

Reply:
also, is this supposed to be a strip ID or sth? in that case, please put that in the name

Reply:
DONE